api/src/main/java/com/flipkart/poseidon/validator/TaskValidator.java

```diff
@@ -37,6 +37,7 @@
 public class TaskValidator {
     public static List<String> validate(Map<String, TaskPOJO> tasks, ParamsPOJO params, Map<String, Class<? extends DataSource<?>>> datasources, boolean validateDataSources) {
         List<String> errors = new ArrayList<>();
+        boolean skipClassFieldValidation = Boolean.valueOf(System.getProperty("skipClassFieldValidation"));
```

Comment:
we can make this property little more structured and semantical like poseidon.validator.unused_fields/context/param...